python_modules/dagster/dagster/_core/definitions/unresolved_asset_job_definition.py

```diff
@@ -289,6 +295,8 @@ def define_asset_job(
             Values that are not strings will be json encoded and must meet the criteria that
             `json.loads(json.dumps(value)) == value`.  These tag values may be overwritten by tag
             values provided at invocation time.
+        metadata (Optional[Mapping[str, RawMetadataValue]]):
```

Comment:
nit: completing the docstring here would be nice. Copy pasted from elsewhere in the codebase:
```
Arbitrary metadata about the job.  Keys are displayed string labels, and values are one of the following: string, float, int, JSON-serializable dict, JSON-serializable list, and one of the data classes returned by a MetadataValue static method.
```

Reply:
I think you need to push up a change that adds this snippet, but aside from that this PR is good to go! :)

Reply:
Pushed the change with the docstring. Thanks!